client/src/main.js

```diff
@@ -37,15 +37,16 @@ Vue.component('page-footer', Pagefooter)
 Vue.component('time-since', TimeSince)
 Vue.component('loader', w1)
 Vue.use(VueClipboard)
+Vue.use(nem2Sdk)
```

Comment:
we can import directly to file when we need it.

Reply:
yea I put this here for an experiment , removed it